lms/envs/test.py

```diff
@@ -269,6 +269,8 @@
 
 ############################## OAUTH2 Provider ################################
 FEATURES['ENABLE_OAUTH2_PROVIDER'] = True
+# don't cache courses for testing
+OIDC_COURSE_HANDLER_CACHE_TIMEOUT = 0
```

Comment:
I'm inclined to keep it open keep it in because there are a few calls to different endpoints (id token and user info) in close succession that will hit the current cache.